torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15711,6 +15736,65 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # Can't find schemas for this operator for some reason
                DecorateInfo(unittest.expectedFailure, 'TestOperatorSignatures', 'test_get_torch_func_signature_exhaustive'),
            )),
+    UnaryUfuncInfo(
```

Comment:
Add a comment explaining what these OpInfos are for and pointing people to the jiterator note

Reply:
done.